eclipse-theia/theia-blueprint
# [js/ts][debug] temporarily add js-debug* extensions

#### What it does

Add the vscode built-i extensions needed to debug in JS/TS. 

Following the example of the upstream Theia example application:

"Due to an issue when publishing the builtin extension-pack for `1.79.0` we need to manually add the `js-debug` and `js-debug-companion` plugins for JavaScript debugging support."

see: 
- https://github.com/eclipse-theia/vscode-builtin-extensions/issues/123
- https://github.com/eclipse-theia/theia/pull/12835

This commit can be reverted when Blueprint uses a newer release of vscode-built-in extension pack, that includes the js-debug* extensions. No such new release exists as of this writing.

#### How to test

Build the Blueprint app and check that extensions `ms-vscode.js-debug` and `ms-vscode.js-debug-companion` are included. 

#### Review checklist


#### Reminder for reviewers
